src/dwarf.cpp

```diff
@@ -49,7 +49,7 @@ std::uint32_t form_length(dw::form f, freader& s) {
             assert(false);
             return 0; // TODO: read NTSB from s
         case dw::form::block:
-            return leb_block();
+            return (std::uint32_t)leb_block();
```

Comment:
I prefer `static_cast` over the old C-style cast, 1) as it's more explicit what's going on, and 2) it's searchable.
```suggestion
            return static_cast<std::uint32_t>(leb_block());
```